Libraries/src/Amazon.Lambda.CloudWatchEvents/TranscribeEvents/TranscribeJobStateChange.cs

```diff
@@ -0,0 +1,23 @@
+namespace Amazon.Lambda.CloudWatchEvents.TranscribeEvents
+{
+    /// <summary>
+    /// This class represents the details of a Transcribe Job State Change sent via EventBridge.
```

Comment:
Is the a URL you can add to where this event type is documented?

Reply:
https://docs.aws.amazon.com/transcribe/latest/dg/monitoring-events.html

Reply:
Sorry I wasn't clear, can you add the URL as part of the NDoc here?